app/containers/Products/ProductCreatorContainer.tsx

```diff
@@ -27,11 +27,14 @@ export const ProductCreator: React.FunctionComponent<Props> = ({
           units: e.formData.units,
           state: 'active',
           parent: [null],
-          requiresEmissionAllocation: e.formData.requiresEmissionAllocation
+          requiresEmissionAllocation: e.formData.requiresEmissionAllocation,
+          includesImportedElectricity: e.formData.includesImportedElectricity,
+          includesExportedElectricity: e.formData.includesExportedElectricity,
+          includesImportedHeat: e.formData.includesImportedHeat,
+          includesExportedHeat: e.formData.includesExportedHeat
```

Comment:
Maybe we should rename those for more clarity (sorry, I should have thought of that before): 
includesImportedElectricity -> addImportedElectricityEmissions
includesExportedElectricity -> substractExportedElectricityEmissions
includesImportedHeat -> addImportedHeatEmissions
includesExportedHeat -> substractExportedHeatEmissions